### ISSUE TITLE
Further Reading is missing

### ISSUE THREAD

**Issue by insideoutclub** [dlang-tour/english#150]
When I go to this URL: https://tour.dlang.org/tour/en/basics/further-reading the content is missing completely.

At the bottom of the page, I get the arrow for the previous and the next topics and an indication that I'm on page 20 out of 20, but that's all.

**Comment by insideoutclub** [comment]
Sebastian,

I currently can't reach tour.dlang.org. I can reach dlang.org.

Thanks,
Dave

**Comment by wilzbach** [comment]
> I currently can't reach tour.dlang.org. I can reach dlang.org.

Thanks for letting me know (I do get the down-time notifications).
Unfortunately I am only a contributor to the tour and not its creator.
This means while I do have merge access for the projects here at `dlang-tour`, I don't have access to the deployed server - only @stonemaster does.
What I can say is:
- there has been a [recent outage](http://forum.dlang.org/post/vdtbkvoahhsoktcijanh@forum.dlang.org) last week which was due to a Docker problem
- Travis builds the tour and checks for errors (merging is blocked one failures)
- I just tried to build the tour locally and it still works fine

->  it's a deployment problem :/
